### ISSUE TITLE
Centralize issues/github templates and CONTRIBUTING.md

### ISSUE THREAD

**Issue by mkurz** [playframework/play-meta#215]
Probably a good idea to move most (all?) of them into the `.github` repo ([we have there one already](https://github.com/playframework/.github/blob/main/CONTRIBUTING.md)):

* https://github.com/playframework/anorm/blob/main/CONTRIBUTING.md
* https://github.com/playframework/playframework/blob/main/CONTRIBUTING.md
* https://github.com/playframework/play-mailer/blob/main/CONTRIBUTING.md
* https://github.com/playframework/play-slick/blob/main/CONTRIBUTING.md
* https://github.com/playframework/twirl/blob/main/CONTRIBUTING.md

Same with the issue an pull request templates:
* https://github.com/playframework/play-json/tree/main/.github/
* https://github.com/playframework/anorm/tree/main/.github/
* https://github.com/playframework/playframework/tree/main/.github/
* https://github.com/playframework/play-meta/tree/main/.github/
* https://github.com/playframework/play-ws/tree/main/.github/

For issues I think we should just use a `ISSUE_TEMPLATE` folder, we can influence sorting by numbering the files, e.g. [see here](https://github.com/tensorflow/tensorflow/tree/f3fd82f65724cdba600fdd23d251c2b01152ed3c/.github/ISSUE_TEMPLATE).

More information:
* https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file
* https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors
* https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates

**Comment by ihostage** [comment]
I agree 👍 I had the same thoughts. Moreover, I think we also should unify a labels for all repos 😄

**Comment by mkurz** [comment]
Same thought here too about labels ;) I am looking into mergify config right now (after drinking coffee with java beans)

**Comment by mkurz** [comment]
@ihostage So mergify can not read configs from the `.github` repo yet and also can not extend other configs :cry:. I opened a pull request to at least read configs from the `.github` repo: https://github.com/Mergifyio/mergify-engine/pull/4050 I did not implement `_extends` however (yet :wink:)

**Comment by ihostage** [comment]
@mkurz Yep, I explored Mergify opportunities too and know that he cannot do that 😞 
And it's great that you tried to implement this 👍

**Comment by mkurz** [comment]
@ihostage Please vote up this discussion as well: https://github.com/Mergifyio/mergify-engine/discussions/2205 thanks!
